docs/mine/lotus/split-markets-miners.md

```diff
@@ -10,7 +10,9 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 ## Background
 
-> Note: this feature is available starting with Lotus v1.11.1
+::: warning
+This feature is avaiable in lotus v1.11.1-rc and up, however, it is still experimental and under testing. Please use at your own risk.
+::: warning
```

Comment:
[Looks like](https://v1.vuepress.vuejs.org/guide/markdown.html#default-title) only `:::` is used to close the warning block. Right now the [whole page](https://deploy-preview-988--filecoin-docs.netlify.app/mine/lotus/split-markets-miners/#frontmatter-title) is wrapped in a nested warning. 
```suggestion
:::
```